packages/ui/.env.example

```diff
@@ -1,31 +0,0 @@
-# TESTNET Endpoints
```

Comment:
Thank you for removing some of the unnecessary changes but now this file was removed for no reason. Please restore it.

Reply:
fixed it, please review